src/main/scala/mesosphere/marathon/core/scheduler/SchedulerPluginRunner.scala

```diff
@@ -0,0 +1,24 @@
+package mesosphere.marathon
+package core.scheduler
+
+import mesosphere.marathon.plugin.RunSpec
+import mesosphere.marathon.plugin.scheduler.SchedulerPlugin
+import org.apache.mesos.Protos.Offer
+
+/**
+  * Runs external scheduler plugins
+  *
+  * @author Dmitry Tsydzik.
+  * @since 6/15/17.
+  */
+object SchedulerPluginRunner {
+  /**
+    * Is populated by [[mesosphere.marathon.core.scheduler.SchedulerPluginConfigurer]]
+    */
+  var plugins: List[SchedulerPlugin] = List.empty
```

Comment:
I also dislike this approach. But I don't see another way to get list of plugins here. Can you suggest how to fix it?